corehq/apps/analytics/tasks.py

```diff
@@ -144,6 +144,8 @@ def _send_form_to_hubspot(form_id, webuser, cookies, meta):
             url,
             data=data
         )
+        logger.debug('Sent this data to HS: %s \nreceived: %s'
+                     .format(data, json.dumps(json.loads(req.json), indent=2, sort_keys=True)))
```

Comment:
Also, the  `%s` and `format` are not compatible. For logging you should use `%s` and then just include the parameters as arguments to the log call.
```python
logger.warning('Protocol problem: %s', 'connection reset')
```